pkg/manifests/manifests.go

```diff
@@ -582,6 +582,10 @@ func (f *Factory) AlertmanagerMain(trustedCABundleCM *v1.ConfigMap) (*monv1.Aler
 		a.Spec.Resources = *f.config.ClusterMonitoringConfiguration.AlertmanagerMainConfig.Resources
 	}
 
+	if f.config.ClusterMonitoringConfiguration.AlertmanagerMainConfig.Secrets != nil {
+		a.Spec.Secrets = append(a.Spec.Secrets, f.config.ClusterMonitoringConfiguration.AlertmanagerMainConfig.Secrets...)
```

Comment:
Would this append by any case add duplicate entries to AlertManager when the reconcile loop start again?

Reply:
I didnt see issue like that

Reply:
The Alertmanager object is read from the assets directory (it's not the "live" object retrieved from the Kube API).

Reply:
Aah! thanks for the explanation @simonpasquier 🙇